src/App.css

```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body, input, textarea, select, button {
+    font-family: 'Poppins', sans-serif;
+}
+
+.fruit-counters {
+    width: 500px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1em;
```

Comment:
Cool dat je `em` gebruikt! Zet dan wel even in de html-declaratie wat 1em moet voorstellen:

`html { font-size: 14px; }` (of 16, whatever you like)